### ISSUE TITLE
Config directory not created on first use

### ISSUE THREAD

**Issue by james-callahan** [drGrove/mtls-cli#30]
From a fresh install/new user:
```
$ mtls config name "Some Name"
Usage: mtls [OPTIONS] COMMAND [ARGS]...
Try 'mtls --help' for help.

Error: Invalid value for '--config' / '-c': Path '/home/james/.config/mtls/config.ini' does not exist.
```

**Comment by james-callahan** [comment]
You now need to run `mtls init` first: I was hoping that `mtls config` wouldn't need the config file to exist.

**Comment by drGrove** [comment]
The check on the flag was removed, but you're right. If it cannot find a config file when running `mtls config`, we have 2 options.

1) Create  the file and just add the key/value pair.
2) Create the file, run through the init and then add the new key/value pair

**Comment by james-callahan** [comment]
> Create the file and just add the key/value pair.

This sounds like the correct answer

> Create the file, run through the init and then add the new key/value pair

If you wanted `init` you'd run `init`.

**Comment by drGrove** [comment]
Fixed with 23f0b9f9393aba42c04a4ff09d935063a41ecdad
